javascripts/helper.js

```diff
@@ -0,0 +1,87 @@
+console.log ("hello there!");
```

Comment:
During development, console logs should be your best friend - however, once your code moves into production, you'll want to trim out console logs and any non-essential comments throughout the codebase.